src/mixins/behaviors.js

```diff
@@ -70,12 +70,12 @@ export default {
     _.invoke(this._behaviors, 'undelegateEntityEvents');
   },
 
-  _destroyBehaviors: function(args) {
+  _destroyBehaviors: function(options) {
     // Call destroy on each behavior after
     // destroying the view.
     // This unbinds event listeners
     // that behaviors have registered for.
-    _.invoke(this._behaviors, 'destroy', args);
+    _.invoke(this._behaviors, 'destroy', options);
```

Comment:
Actually I think `args` is more correct than options and this should be `...args`  options is most likely `args[0]`
This is getting called from within `destroy: function(...args) {` and passed args, so those arguments would get spread for the destroy events on the behavior event.